example/hyperparam/hyperopt/search_keras.py

```diff
@@ -0,0 +1,104 @@
+import click
```

Comment:
Add a small doc comment at the top of each file to say what it does. It makes it easier to understand than scrolling down into the file. See some of the other examples for reference.